tests/migration_test.go

```diff
@@ -688,7 +684,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 						},
 					},
 				}
-				vmi = runVMIAndExpectLaunch(vmi, 240)
+				vmi = tests.RunVMIAndExpectLaunch(vmi, 240)
```

Comment:
Maybe we can also change the literal `240` to some constant?
Perhaps such a constant exists already?

Reply:
Not that I am aware of. I have these magic numbers on my todo list. There is plenty of things we can improve :)